x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -276,7 +280,7 @@ export class TaskManagerRunner implements TaskRunner {
     }
     this.logger.debug(`Running task ${this}`);
 
-    const apmTrans = apm.startTransaction(this.taskType, 'taskManager run', {
+    const apmTrans = apm.startTransaction(this.taskType, TASK_MANAGER_RUN_TRANSACTION_TYPE, {
       childOf: this.instance.task.traceparent,
     });
```

Comment:
Should we add `apmTrans?.addLabels({ ephemeral: false });` here?

Reply:
I think it's kind of redundant, no :-) ?
As you pefer!

Reply:
I think it's fine without!